ui/ducks/send/send.js

```diff
@@ -260,7 +267,7 @@ async function estimateGasLimitForSend({
   }
 }
 
-async function getERC20Balance(token, accountAddress) {
+export async function getERC20Balance(token, accountAddress) {
```

Comment:
I'm not seeing this method used outside of this file?

Reply:
its being exported so it can be mocked in tests

Reply:
Ah gotcha. I didn't realize that was necessary. Cool